### ISSUE TITLE
Source Code in footer should point to core software

### ISSUE THREAD

**Issue by jermanuts** [monero-project/monero-site#2401]
In https://getmonero.org/ footer "Source Code" it points to https://github.com/monero-project/monero-site which is uncommon behavior, it should point to https://github.com/monero-project/monero. People are probably looking for the actual Monero software source code rather the website.

**Comment by mosin311aa** [comment]
No, quite the opposite actually. The website existing does not correlate to Monero existing or not, it is merely a place for information and it should be kept that way. Pointing it to the direct source code will cause confusion. I don't know about other people, but when I clicked that hyperlink 2 minutes ago, I was expecting to be redirected to the monero-site source code and not to the actual Monero source. I can find that elsewhere. 

Keep the site seperate.
